### PR TITLE
Fix operational template parsing 2

### PR DESCRIPTION
**Pull request by pieterbos**
Fix operational template parsing, by allowing both ```component_terminologies <...>``` and ```component_terminologies component_terminologies = <...>``` syntax

### CODE REVIEW

File: aom/src/main/java/com/nedap/archie/adlparser/treewalkers/ComponentTerminologiesHelper.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package com.nedap.archie.adlparser.treewalkers;
+
+import com.nedap.archie.aom.terminology.ArchetypeTerminology;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * A helper class for parsing component terminologies. Not to be used outside of the parser
+ */
```
Comment (steijgeler) [inline aom/src/main/java/com/nedap/archie/adlparser/treewalkers/ComponentTerminologiesHelper.java]: So make it package-private? This obviously isn't possible if the parser consists of multiple packages, but I am not sure if it is

**Comment by pieterbos** [comment]
example 2 is what the ADL workbench currently outputs. Example 1 is what we used to output, and example 3 is what Thomas Beale suggested as a better syntax that he is willing to implement. So we just accept all thee now, so people don't run into problems. Example 1 is only for backwards compatibility but since it is 100% valid odin it's hard to get rid of.

File: aom/src/main/java/com/nedap/archie/adlparser/treewalkers/ComponentTerminologiesHelper.java (hunk 1 above)
Comment (pieterbos) [reply aom/src/main/java/com/nedap/archie/adlparser/treewalkers/ComponentTerminologiesHelper.java]: I'll change it.

**Comment by wolandscat** [comment]
This is the correct syntax, according to me now (actually just Pieter's original, which I think is more systematic than what the ADL WB was doing). So the following is straight from the ADL WB now.

```
component_terminologies
	["openehr-task_planning-DECISION_GROUP.determine_er_positive_hormone_regime.v0.0.1"] = <
		term_definitions = <
			["en"] = <
				["id1"] = <
					text = <"Menstrual status (BPM node 21)">
					description = <"-">
				>
			>
		>
	>
	["openehr-task_planning-DECISION_GROUP.determine_tamoxifen_regime.v0.0.1"] = <
		term_definitions = <
			["en"] = <
				["id1"] = <
					text = <"Determine Tamoxifen regime  (BPM node 24)">
					description = <"-">
				>
```

**Pull request merged**